.vscode/settings.json

```diff
@@ -39,5 +39,20 @@
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
     "editor.tabSize": 4,
-    "editor.insertSpaces": true
+    "editor.insertSpaces": true,
+    "cSpell.words": [
+        "affordances",
+        "alparser",
+        "bauth",
+        "blockwise",
+        "Coap",
+        "Coaps",
+        "mcollina",
+        "nosec",
+        "observeproperty",
```

Comment:
I need to add "unobserverproperty" as well to avoid accidental replacement of unobserverproperty by observerproperty by the spelling checker !